README.md

```diff
@@ -3,11 +3,24 @@
 
 ## ⚠️ Under Construction  ⚠️
 We are currently working hard to be ready for a stable release.
+  - Please *don't share* this repo until we make our public announcement!
+  - If you are not already part of our Beta program and would like to join, please make a [request here](https://github.com/adobe-private/react-spectrum-v3/issues/new?template=Feedback.md).
+  - Please try out our RC React Spectrum Components and [tell us](https://github.com/adobe-private/react-spectrum-v3/issues/new/choose) what you think!
 
-  - Please *don't share* this repo until we make our announcement!
-  - Please try out our rc components and tell us what you think!
+## Overview ##
+React Spectrum is a UI Component Library leveraging Spectrum, Adobe's Design System. It will enable you to easily build apps with plug and play UI components that match Adobe's look and feel. Some features of React Spectrum include: 
```

Comment:
Small nitpick as per documentation tensing:
"will enable you" -> "enables you" or "helps you"